packages/components/src/draggable/index.js

```diff
@@ -17,7 +17,7 @@ const bodyClass = 'is-dragging-components-draggable';
 
 /**
  * @typedef Props
- * @property {(props: RenderProp) => JSX.Element | null} children Children.
+ * @property {(props: RenderProp) => JSX.Element | null} children Children. TODO: any action needed here?
```

Comment:
`draggable` is currently outside of the `ViewOwnProps` context system so we don't need to worry about it.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/31817/commits/e250f060a87d5ecfb483d9cdd08c7e4d35df6d3e